Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -140,15 +179,6 @@ public void EndActiveRound()
         roundTimer.OnTimerUpdate -= HUDTimerUpdate;
         roundTimer.OnTimerRunCompleted -= EndActiveRound;
         AssignRewards();
-
```

Comment:
This should not have been moved from matchManager into TownScoreBoard. The reason it was structured this way initialy was to ensure the all functions subscribed to the onRoundEnd delegate were invoked before we changed the scene. 

Now important functions like the one that decouples and transfers playerInput isn't able to be executed before players scene is changed. This leads to the bidding scene being broken.